src/scripts/tweaks/restore_attribution_links.js

```diff
@@ -3,13 +3,11 @@ import { filterPostElements } from '../../util/interface.js';
 import { keyToCss } from '../../util/css_map.js';
 import { timelineObject } from '../../util/react_props.js';
 
-const timeline = /\/v2\/timeline\/dashboard/;
-
 const postAttributionSelector = keyToCss('postAttribution');
 const reblogAttributionSelector = keyToCss('reblogAttribution');
 
 const processPosts = async function (postElements) {
-  filterPostElements(postElements, { timeline }).forEach(async postElement => {
+  filterPostElements(postElements).forEach(async postElement => {
```

Comment:
with no arguments, this becomes synonymous with...

```suggestion
  postElements.forEach(async postElement => {
```

Reply:
Does it? Looks like filterPostElements is still doing some work:

```
postElements = postElements.map(element => element.closest(postSelector)).filter(Boolean);
```

But if that work is not necessary here, then I can drop the function call.

Reply:
Ah, I see, postSelector is already used in the onNewPosts logic.